libavfilter/dnn_filter_common.c

```diff
@@ -113,8 +125,11 @@ DNNReturnType ff_dnn_get_input(DnnContext *ctx, DNNData *input)
 
 DNNReturnType ff_dnn_get_output(DnnContext *ctx, int input_width, int input_height, int *output_width, int *output_height)
 {
-    return ctx->model->get_output(ctx->model->model, ctx->model_inputname, input_width, input_height,
-                                    (const char *)ctx->model_outputnames[0], output_width, output_height);
+    DNNBackendType backend = ctx->backend_type;
```

Comment:
why pytroch is so speical ?

Reply:
> why pytroch is so speical ?

Since pytorch C++ api do not need input/output names to do the inference.